.gitignore

```diff
@@ -1,13 +1,38 @@
-target/
-**/target/
+!polkadot.service
+.cargo-remote.toml
+.direnv/
+.DS_Store
+.env
```

Comment:
```suggestion
.env*
```
Some `.env*` files can be sensitive so I would recommend being "aggressive" here by default.